templates/discord-activities/react-colyseus-ts/package.json

```diff
@@ -40,6 +40,7 @@
 		"colyseus.js": "^0.15.18",
 		"express": "^4.19.2",
 		"react": "^18.2.0",
+		"react-colyseus-ts": "file:",
```

Comment:
Would you mind removing these `package.json` changes? Not sure how they got there, but it may break our CI pipeline.